src/wrappers/ti_common_wrapper.cpp

```diff
@@ -290,9 +291,32 @@ std::map<std::string, expected_file_t> ti_common_wrapper_t::get_build_files() {
 }
 
 void ti_common_wrapper_t::append_response_file(const std::string& response_file) {
-  auto args_string = file::read(response_file);
-  std::replace(args_string.begin(), args_string.end(), '\n', ' ');
-  m_resolved_args += string_list_t::split_args(args_string);
+  std::ifstream stream(response_file);
```

Comment:
The disadvantage with using `std::ifstream` directly like this (instead of `file::read()`) is that you miss out on cross-OS Unicode support. On Windows this will most likely *not* work with paths that include characters that are not part of the Windows [CP-1252 character set](https://en.wikipedia.org/wiki/Windows-1252#Character_set).

Would it be possible to solve the same problem with `file::read()` and a `string_list_t` instead? E.g. something like this:

```c++
  string_list_t lines(file::read(response_file), "\n");
  for (auto& line : lines) {
    ...
    m_resolved_args += string_list_t::split_args(line);
  }
```

Reply:
Ok, did not think about that! I made a second attempt now (892e6b9e453b010d7346da993037d60d3ad83f7c), according to your suggestion.

Reply:
Great! Except now `<fstream>` is no longer needed.

Reply:
Right! Fixed.